modin/data_management/data_manager.py

```diff
@@ -275,6 +275,28 @@ def inter_data_op_builder(left, right, self_cols, other_cols, func):
         new_data = reindexed_self.inter_data_operation(1, lambda l, r: inter_data_op_builder(l, r, self_cols, other_cols, func), reindexed_other)
 
         return cls(new_data, joined_index, new_columns)
+
+    def _operator_helper(self, func, other, axis, level, *args):
+        """Helper method for inter-DataFrame and scalar operations"""
+        if level is not None:
+            raise NotImplementedError("Mutlilevel index not yet supported "
+                                      "in Pandas on Ray")
+
+        if hasattr(other, "_data_manager"):
+            return self.inter_manager_operations(other._data_manager, "outer",
+                lambda x, y: func(x, y, axis, level, *args))
+        else:
+            return self.scalar_operations(axis, other,
+                lambda df: func(df, other, axis, level, *args))
+
+    def add(self, **kwargs):
+        func = pandas.DataFrame.add
```

Comment:
I'll probably move these `kwargs.get()` statements to go inside `_operator_helper()`